scripts/run-confluence-container.sh

```diff
@@ -32,18 +32,24 @@ esac
 # Set current folder to parent
 cd "$(dirname "$0")"/..
 
-if [[ ! -z "${CONFLUENCE_VERSION}" ]]
+#load default env varibles
+set -o allexport
```

Comment:
This loads the default env variables. 
Is needed to make the database variable available for the run